python/osrd_schemas/osrd_schemas/rolling_stock.py

```diff
@@ -138,6 +189,7 @@ class RollingStock(BaseModel, extra=Extra.forbid):
     rolling_resistance: RollingResistance = Field(description="The formula to use to compute rolling resistance")
     loading_gauge: LoadingGaugeType
     metadata: Mapping[str, str] = Field(description="Properties used in the frontend to display the rolling stock")
+    energy_sources: Optional[List[EnergySource]]
```

Comment:
Instead of an optional list can we have a default empty list? This implies adding this empty list to your migration.